From 4c645ff41b074f5ce646f6d231badfc1d06a3556 Mon Sep 17 00:00:00 2001
From: Derek Kozel <derek.kozel@ettus.com>
Date: Mon, 28 Mar 2016 17:35:39 -0700
Subject: [PATCH 07/12] Removed mention of OSX PPC support

OS X 10.5.8 is the last release to support PPC (current is 10.11.3) and
the default compilers in that release do not meet UHD's minimum
requirements. Also various dependency versions are not met.
---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 76d34d0..1aa57bc 100644
--- a/README.md
+++ b/README.md
@@ -30,7 +30,7 @@ UHD is primarily developed on Linux, but we also test and support the following
 operating systems.
 
 * Linux (Fedora and Ubuntu)
-* Mac OS X (PPC and Intel)
+* Mac OS X (Intel)
 * Windows 7
 
 Other operating systems will most likely work, too, but are not officially
-- 
2.1.4

